spring-cloud-starter-stream-processor-image-recognition/src/main/java/org/springframework/cloud/stream/app/image/recognition/processor/ImageRecognitionTensorflowOutputConverter.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.image.recognition.processor;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.tensorflow.Tensor;
+
+import org.springframework.cloud.stream.app.tensorflow.processor.TensorflowOutputConverter;
+import org.springframework.core.io.Resource;
+import org.springframework.tuple.Tuple;
+import org.springframework.tuple.TupleBuilder;
+import org.springframework.util.Assert;
+import org.springframework.util.StreamUtils;
+
+import com.google.common.collect.Ordering;
+import com.google.common.primitives.Floats;
+
+/**
+ * @author Christian Tzolov
+ */
+public class ImageRecognitionTensorflowOutputConverter implements TensorflowOutputConverter<Tuple> {
+
+	private static final Log logger = LogFactory.getLog(ImageRecognitionTensorflowOutputConverter.class);
+
+	private final List<String> labels;
+
+	private final int responseSize;
+
+	public ImageRecognitionTensorflowOutputConverter(Resource labels, int responseSize) {
+		this.responseSize = responseSize;
+		try (InputStream is = labels.getInputStream()) {
+			this.labels = Arrays.asList(StreamUtils.copyToString(is, Charset.forName("UTF-8")).split("\n"));
```

Comment:
Looks like that `\n` label delimiter must be mentioned in docs as well.

Reply:
+1